modules/issue_tracker/php/module.class.inc

```diff
@@ -24,6 +27,37 @@ namespace LORIS\issue_tracker;
  */
 class Module extends \Module
 {
+    /**
+     * The default handler for a module routes to a page of the first component of
+     * the URL's name, or the same name as the module if serving the root of the
+     * module.
+     *
+     * Modules extend PrefixRouter, so that the parent handler knows that the
+     * css, static, and js routes should be routed through the file system to
+     * retrieve the files before trying to load any specific pages.
+     *
+     * @param ServerRequestInterface $request The incoming PSR7 request
+     *
+     * @return ResponseInterface the outgoing PSR7 response
+     */
+    public function handle(ServerRequestInterface $request) : ResponseInterface
+    {
+        $path = trim($request->getURI()->getPath(), '/');
+        switch ($path) {
+        case 'Attachment':
+            \Module::factory('api');
+            $handler = new endpoints\Attachment();
+            break;
+        default:
+            return parent::handle($request);
+        }
+        $user = $request->getAttribute('user');
+        if (!$handler->_hasAccess($user)) {
```

Comment:
`_` prefixed function are supposed to be private methods.
It is fine for it to be private but it will need to be called from the handle function of LORIS\issue_tracker\endpoints\Attachment.
The alternative is to have this Module class have the _hasAccess function.